tests/pyreverse/test_writer.py

```diff
@@ -28,7 +28,7 @@
 from unittest.mock import Mock
 
 import pytest
-from conftest import PyreverseConfig  # type: ignore
+from conftest import PyreverseConfig  # type: ignore #pylint: disable=no-name-in-module
```

Comment:
For some reason the import of `conftest` introduced in https://github.com/PyCQA/pylint/commit/25000863761220c1aa5db129e3cd41f5fe51b167 failed the `pre-commit`.

Reply:
Shouldn't this be `from pylint.pyrevers.conftest import PyreverseConfig`?

Reply:
See discussion here: https://github.com/PyCQA/pylint/pull/4950#discussion_r702048576
Seems like we can't fix this without a (big) refactor